### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by blemia51**

### CODE REVIEW

File: back/index.js [hunk 1]
```diff
@@ -1,11 +1,42 @@
...
+
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  next();
+});
+
 app.get('/', (request, response) => {
   response.send('Bienvenue sur Express');
 });
```
Comment (rocambille) [inline back/index.js]: request/response or req/res, regular or arrow function there are the questions :)

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
make separate PRs for front and back please

File: back/index.js (hunk 1 above)
Comment (blemia51) [reply back/index.js]: done

**Comment by blemia51** [comment]
no more needed

**Pull request closed without merging**